### PR TITLE
Refactor positioning

### PR DESCRIPTION
**Pull request by TimSpeelman**
Solves #325

### CODE REVIEW

File: TriblerGUI/widgets/trustpage/js/ideas.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Positioning
```
Comment (TimSpeelman) [inline TriblerGUI/widgets/trustpage/js/ideas.txt]: Please ignore this file, already removed

File: TriblerGUI/widgets/trustpage/js/radial_view/radial_positioning.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+            if (new_node) {
+                new_node.x = node.x;
+                new_node.y = node.y;
+            }
+        });
+
+        // Position all new nodes at the focus node
+        newGraphData.nodes.forEach(function (node) {
+            if (!('x' in node)) {
```
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_positioning.js]: Why do you only check for "x" if y is set afterwards as well?

Comment (TimSpeelman) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_positioning.js]: because they are always set together, checking if y is set would be redundant

Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/radial_view/radial_positioning.js]: Fair enough

**Pull request merged**